Recrutify/Recrutify.Host/Controllers/SchedulesController.cs

```diff
@@ -30,5 +34,28 @@ public Task<ScheduleDTO> GetByDatePeriodForCurrentUserAsync([FromQuery] DateTime
         {
             return _scheduleService.GetByDatePeriodForCurrentUserAsync(date, daysNum);
         }
+
+        [Authorize(Policy = Constants.Policies.AllAccessPolicy)]
+        [HttpPut]
+        public async Task<ActionResult> UpdateScheduleSlotsAsync(IEnumerable<DateTime> dates)
+        {
+            try
+            {
+                await _scheduleService.UpdateScheduleSlotsForCurrentUserAsync(dates);
+            }
+            catch (NotFoundException)
```

Comment:
You don't throw not found anywhere, so remove this catch

Reply:
+